src/Symfony/Bridge/PhpUnit/composer.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "symfony/phpunit-bridge",
+    "type": "symfony-bridge",
+    "description": "Symfony PHPUnit Bridge",
+    "keywords": [],
+    "homepage": "http://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Nicolas Grekas",
```

Comment:
As a matter of fact, I'm the author of this component... @fabpot is the copyright owner mentionned in the LICENSE file, but I wrote the code. This is just plain descriptive truth. If someone else wrote some other component (or a signficant nontrivial part of it), he/she should also be mentionned as author IMHO.